services_app/src/main/java/org/opendatakit/services/sync/actions/fragments/LoginFragment.java

```diff
@@ -453,8 +455,15 @@ private void performActionOnSyncComplete(SyncStatus syncStatus){
    }
 
    public void showAlertDialog(String title, String message, DialogInterface.OnClickListener onPositiveButtonClick){
-      AlertDialog alertDialog = new AlertDialog
-              .Builder(requireActivity())
+
+      /**
+       * New dialog styling
+       * MaterialAlertDialogBuilder is standard for all ODK-X Apps
+       * OdkAlertDialogStyle present in AndroidLibrary is used to style this dialog
+       * @params change MaterialAlertDialogBuilder to AlertDialog.Builder in case of any error and remove R.style... param!
+       */
+
+      AlertDialog alertDialog = new MaterialAlertDialogBuilder(requireActivity(),R.style.OdkXAlertDialogStyle)
```

Comment:
Why isn't the type simply MaterialAlertDialogBuilder instead of androidx.appcompat.app.AlertDialog?